Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty_test.py

```diff
@@ -0,0 +1,25 @@
+from AWSGuardDuty import get_members
+
+RESPONSE = {
+    'Members': [
+        {},
+        {
+            'AccountId': 1,
+            'DetectorId': 1,
+            'MasterId': 1,
+        }
+    ]
+}
+
+
+class Client:
+    @staticmethod
+    def get_members(DetectorId, AccountIds):
+        return RESPONSE
+
+
+def test_get_members(mocker):
```

Comment:
please add a docstring (given/when/then) to define what are you testing (the bug fix)

Reply:
done